### PR TITLE
PERFTEST: Fix warmup iters to be non-zero

### PR DESCRIPTION
**Pull request by bureddy**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10752/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 2 of 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 28-Jun-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/0f7a8913d7717b5e3e38c0d4bd9c3b72318dae88/_main) | :x: FAILURE
[hpc-test-node2_W2](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/713da67f966781d5f4d4b6f1d52e0215b1e31331/hpc-test-node2_W2) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/1f016457a1a975bdaf48922ddf0ce3f5a1ebb577/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/ad7730549ae90e02e9a0ea003136c35c5b215a9a/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/9603cf57b3bb691c0abc442543626c076a93efc5/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/148092e463cc9fdca51575a4ca3a6a60654bb6a9/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/d801b2f720691a3a04a2bb04aef45c5d4829882e/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/f83c31cbfd1b00d7f655632d6243b0e5fb6ad6ef/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/6f5b140ab44fa268752bd4a8c705221eaf5516ba/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/807a04be6f1543f873b5dff14d17f2169017b6b3/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/74336d62fa343b1859137046f11f93c26fd6d4a0/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/1e20e2c06040deee82991a1027cc6f45cfc73dcb/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/62c4a737b5551bbdd760c94dcee9da7fb48d8cac/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/86a54f412dd022f1c30fdb90a402f101ede8495c/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W0](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/33883d45f14560748cb99597b74b8d605efda681/hpc-test-node2_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W1](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/7a2b65998579fbd179ab74cfc244ee5b203474e3/hpc-test-node2_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W3](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/6b5cec3016500ea711aaec1cce3452e440a08578/hpc-test-node2_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W0](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/b3d8f93658d69a9b77707f27d5eb63b90296959c/hpc-test-node3_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W1](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/b7853258d900b161003e7a2dd0a9fca85d195b19/hpc-test-node3_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W2](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/1b2b745bd261330028e37625912ed1b73f99c09e/hpc-test-node3_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W3](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/0db73dac6b402210c7493141cf45c73b013946eb/hpc-test-node3_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node_W0](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/5971d5a1a46d37ba0dbed7d4d90e5f4c8a2a5c34/hpc-test-node_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node_W1](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/93e53b64aafe820af6e3f1596819605d5a4e1e8b/hpc-test-node_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node_W2](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/a0dc597ae47b6f090545a9d97dd0828563ab65ad/hpc-test-node_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node_W3](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/2e5d9560a5e21e72d5ac4910560a073527e309f2/hpc-test-node_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/6530bc21f17a5f1191ed17c10f156e3b0a6c6b89/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/1baf86659fb8f7a83ed5cd82d1d8192c07d8b54b/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/8b5e180d4585c7119eb50ec8b41912466c8019c5/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/5aff38c654b12c39e1688af57b7937a8d4ed99b1/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/4a411e8df3c86cd40b3befc247157c1083c54475/vulcan-jenkins_W0) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/a375817292ede9f11e58dae142339e3d63392c7b/vulcan-jenkins_W1) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/8d6831cea2e4aa4a74adcbc958ce4d326dac18a5/vulcan-jenkins_W2) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/46291c9a379348a346c3b813eb277d4c/raw/d12d0a918bb5cd86402260c8ab9350e1598bfd49/vulcan-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
bot:mlx:retest

**Comment by yosefe** [comment]
bot:pipe:retest

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10757/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 2 of 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 29-Jun-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/08a7da43521421669156e1ab54ce8cfe890bc986/_main) | :x: FAILURE
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/42bfc4f36b4da18495d57ab2f051b81f2ce9e1e7/hpc-arm-hwi-jenkins_W1) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/43a4affd6bf8ad6df639d1e01b521c0196311d2b/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/1e511af491d16746d996569b4008c679e3b36efc/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/9e8e88dc73bead2e50d5c2a5982cd7fe18d5a20a/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/5921c7cea456670537ff52f181103785384d38be/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/434158d97523af7c1f5f5cb0ae328e0639729a0a/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/e209e71009330e40102dd2353bbcf895e0ea51fa/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/d39159958dd65ecdf36d60e70e9ffcabed5f77dc/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/a207c42b40ad1c0f5613625836e7ad4030a51742/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/f8853b6da2982ec0af2bdb8f1d5d45fc91e9c5c8/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/db9edd324c646374deebdeda8a4a838e3d1e9b79/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/be65efde51adeb6793303d7c25d5183f7a6d150e/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W0](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/09ad3a38029f402a10174a94cb70e6888668142e/hpc-test-node2_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W1](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/d9ecf541fca9b318a6c056ab5a1a9cfd2afb71ba/hpc-test-node2_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W2](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/269bb4cec30826a7b398fc7430ae19c512c749e7/hpc-test-node2_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node2_W3](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/9f4e5a1637771647bbc63df7dae0f96af704a67d/hpc-test-node2_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W0](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/d03e9d35ea9f8b08d0abfcc58eeb47020f91270d/hpc-test-node3_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W1](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/9506e38d5f8fab644e11cbc052b9f23f7e857186/hpc-test-node3_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W2](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/a3c4a5c87a5b084eb5a1d3c476fa3e76b727f2bd/hpc-test-node3_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node3_W3](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/55fa8834a9e2969a4da1bee2a6ff3f22c504f809/hpc-test-node3_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node_W0](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/e9cbbdd6722363b01a032a904b28c9e9047a5339/hpc-test-node_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node_W1](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/17cf0d79760d6b22ed0c8312babf682cc1612181/hpc-test-node_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node_W2](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/d28ba59a637e31f15c9ac7b4897cb367c86d1bab/hpc-test-node_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node_W3](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/2dbb87ac0bf38ab3e6a7413cdcb9e2a23167ae16/hpc-test-node_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/2c04460797b5a8a8fb54070424d5007a840ea8b5/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/92d079f586b5cf4fea9fff57cf3c569739863353/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/acaf4398d5e699cc5c52f0e7380c61bb5f71c984/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/66dc543a40ab6dd4c42ab1ab31ae459400669305/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/d6316296c502999782842c1a8bac8bf77930da35/vulcan-jenkins_W0) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/73ddd1482fff524a14c8c66f88b11dcd39462801/vulcan-jenkins_W1) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/d9863104b816630ee5a4a749685efffea6a315a6/vulcan-jenkins_W2) | :heavy_check_mark: SUCCESS
[vulcan-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/1728f1c985964aedb72a907c2b9c1253/raw/65e5961a28ce61418dcf46946d654b687ee1294f/vulcan-jenkins_W3) | :heavy_check_mark: SUCCESS

File: src/tools/perf/lib/libperf.c [hunk 1]
```diff
@@ -1189,7 +1189,7 @@ static void ucp_perf_test_cleanup_endpoints(ucx_perf_context_t *perf)
 
 static void ucx_perf_set_warmup(ucx_perf_context_t* perf, ucx_perf_params_t* params)
 {
-    perf->max_iter = ucs_min(params->warmup_iter, params->max_iter / 10);
+    perf->max_iter = ucs_min(params->warmup_iter, ucs_div_round_up(params->max_iter, 10));
```
Comment (yosefe) [inline src/tools/perf/lib/libperf.c]: @bureddy looks like doing 10 warmup iterations is slowing down the ARM testing..
let's make it 5 instead of 10?

Comment (bureddy) [reply src/tools/perf/lib/libperf.c]: @yosefe this fix would increase only 1 warm iterations from whatever it was before. right? or am i missing something?

**Comment by bureddy** [comment]
bot:mlx:retest

**Comment by bureddy** [comment]
bot:pipe:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10769/ for details (Mellanox internal link).

File: src/tools/perf/lib/libperf.c (hunk 1 above)
Comment (yosefe) [reply src/tools/perf/lib/libperf.c]: right, never mind.

**Pull request merged**